### PR TITLE
Fix add command help text

### PR DESCRIPTION
**Pull request by anupama-pathirage**
Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help [hunk 1]
```diff
@@ -1,16 +1,17 @@
 NAME
-       Ballerina add - Add a Ballerina module in a project
+       ballerina add - Add a new Ballerina module to a project
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help]:
```suggestion
       ballerina add - add a new module to a Ballerina project
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+
 
 SYNOPSIS
-       ballerina add <module-name> [-t|--template <template-name>]
+       ballerina add <module-name> [-t | --template <template-name>]
+                     [-l | --list ]
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help]:
```suggestion
                     [-l | --list]
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help [hunk 3]
```diff
@@ -1,16 +1,17 @@
 NAME
-       Ballerina add - Add a Ballerina module in a project
+       ballerina add - Add a new Ballerina module to a project
+
 
 SYNOPSIS
-       ballerina add <module-name> [-t|--template <template-name>]
+       ballerina add <module-name> [-t | --template <template-name>]
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help]: Is it just me or does this seem like

`-t` 
OR
`--template <template-name>`

Should this be 
`[-t <template-name> | --template <template-name>]` instead?

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
+                     [-l | --list ]
 
 
 DESCRIPTION
-       Ballerina add command will initialize a new Ballerina module
-       inside the Project. The command should be run inside a Ballerina
-       project.
+       Add a new Ballerina module to the project. The command should
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help]:
```suggestion
       Add a new module to a Ballerina project. The command should
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help [hunk 5]
```diff
@@ -1,16 +1,17 @@
...
+       be run inside a Ballerina project.
 
-       You can also select different templates for the module.
+       Also different templates can be selected to create the new module.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help]:
```suggestion
       A template can also be specified when creating the new module.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help (hunk 1 above)
Comment (anupama-pathirage) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help]: Shouldn't it be a valid complete sentence?

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help (hunk 3 above)
Comment (pubudu91) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help]: It seems like we are not following the convention used for these options. Have a look at https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

If an option starts with `--`, it's a long option. And for long options, you give values as `--opt=<value>`. Not `--opt <value>`. Checked in man pages of git, grep etc. and they all follow the same convention

Comment (pubudu91) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help]: Ah nvm. The key=value convention seem to be used when the value is optional.

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help (hunk 1 above)
Comment (MaryamZi) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help]: `man cat` produces something like

```
$ man cat
NAME
       cat - concatenate files and print on the standard output
```

If it starts with an upper case letter we have to finish with a full-stop too?

**Pull request merged**